Detectors/TRD/base/include/TRDBase/TRDSimParam.h

```diff
@@ -80,8 +80,37 @@ class TRDSimParam
   bool CTOn() const { return mCTOn; }
   bool TimeStructOn() const { return mTimeStructOn; }
   bool PRFOn() const { return mPRFOn; }
-  double TimeResponse(double time) const;
-  double CrossTalk(double time) const;
+  inline double TimeResponse(double time) const
```

Comment:
This is ok. Can you please just separate declaration and definition? Please put the actual implementation below the class in the header.

Reply:
Thanks for the advise. I have moved the implementation after the class definition.